packages/shadcn/src/components/ui/dialog.tsx

```diff
@@ -0,0 +1,100 @@
+'use client';
+
+import * as React from 'react';
+import * as DialogPrimitive from '@radix-ui/react-dialog';
```

Comment:
Rather than use this, import each of the things you need directly from `react` and `@radix-ui/react-dialog`